apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/(analysis)/summary/components/SummaryMetadata.tsx

```diff
@@ -13,10 +13,27 @@ const StatCard = ({ label, percentage, value, tooltipText }) => (
     <Tooltip>
       <TooltipTrigger>
         <div className="flex h-full cursor-default flex-col justify-between space-y-2 rounded-xl border border-slate-200 bg-white p-4 text-left shadow-sm">
-          <p className="text-sm text-slate-600">
+          <p className="flex items-center gap-1 text-sm text-slate-600">
             {label}
-            {percentage && percentage !== "NaN%" && (
-              <span className="ml-1 rounded-xl bg-slate-100 px-2 py-1 text-xs">{percentage}</span>
+            {typeof percentage === "number" && !isNaN(percentage) && (
+              <>
+                <span className="ml-1 rounded-xl bg-slate-100 px-2 py-1 text-xs">{percentage}%</span>
+                {percentage > 100 && (
+                  <TooltipProvider delayDuration={50}>
+                    <Tooltip>
+                      <TooltipTrigger>
+                        <InfoIcon className="h-4 w-4 text-slate-600" />
+                      </TooltipTrigger>
+                      <TooltipContent>
+                        <p>
+                          The percentage is greater than 100% because the responses were created via API and
```

Comment:
We don't know for sure if the responses were created through the API why I'm not fully satisfied with the tooltip. We have a special condition in the StatCards only showing a percentage when there are displays, e.g.
```
        <StatCard
          label="Starts"
          percentage={displayCount === 0 && totalResponses > 0 ? null : Math.round(startsPercentage)}
          value={totalResponses === 0 ? <span>-</span> : totalResponses}
          tooltipText="Number of times the survey has been started."
        />
```

I think I would prefer not showing a percentage at all if the percentage is above 100.

Having users creating responses via the API is a very very small percentage of users and they might know that something is off in the way they use Formbricks.

Reply:
@mattinannt, Is there any possible way to create a response other than API and survey display?
@jobenjada, what do you think about this?